bindings_generator/api.json

```diff
@@ -38259,7 +38259,7 @@
 			},
 			{
 				"name": "get_direct_space_state",
-				"return_type": "PhysicsDirectSpaceState",
+				"return_type": "Object",
```

Comment:
I know the reason for this change, but in a way this is not a good change to make in this file here, I'd probably add a special case to the generator instead, since a user might wants to swap out this file in case she uses custom modules etc